how-to/TOC.md

```diff
@@ -14,3 +14,8 @@ private: true
 ## [Manage the SQL Servers you want to monitor](manage-servers-want-monitor.md)  
 # Server Environment Documentation
 ## [Create Server Environment Documentation](create-server-environment-documentation.md)
+# Compare Wizard
+## [Compare Entire Snapshots](compare-snapshots.md)
+## [Single SQL Server](compare-single-sql-server.md)
+## [Databases](compare-databases.md)
+## [Security](compare-security.md)
```

Comment:
zašto se ovo zove security, a u članku je title: compare users and roles

Reply:
popravila thx